third_party/uvwasi/CMakeLists.txt

```diff
@@ -42,6 +42,10 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
   list(APPEND uvwasi_defines _GNU_SOURCE _POSIX_C_SOURCE=200112)
 endif()
 
+if (WALRUS_MODE STREQUAL "debug")
+    list(APPEND uvwasi_defines UVWASI_DEBUG_LOG)
```

Comment:
Does debug log contain -O0 and -g ?

Reply:
I just checked; it does not, uvwasi only had them enabled when coverage reporting is enabled. Updated the patch so they are also enabled when walrus is in debug mode.

Reply:
It seems that it should be added with `-DUVWASI_DEBUG_LOG` to turn on this option.
As the uvwasi_defines list is directly added by target_compile_definitions below
`target_compile_definitions(uvwasi_a PRIVATE ${uvwasi_defines})`

Or what about using `USWASI_DEBUG_LOG` option defined in CMakeList.txt ?